api/scripts/elastic_ops/get_analyst_tif_files/main.py

```diff
@@ -0,0 +1,291 @@
+from elasticsearch import Elasticsearch, exceptions as es_exceptions
```

Comment:
Added this on this branch as part of finding geotiffs from deployed Dojo elasticsearch. Though it is unrelated, will merge in as it is useful and doesn't affect the rest of the PR changes.